README.md

```diff
@@ -1,6 +1,28 @@
 # Bright_events
 A web application that provides a platform for event organizers to create and manage different types of events. 
 
+**Application Features**
+
+*Creating Events
+*Events RSVP 
+
 A user can create, view, edit and delete events. Other users can also the events posted and can respond to them (RSVP). The event owner is able to view people who have responded to their events,thus improve planning.
 
-Events can be grouped according to location or categories and users can also search events based on the same.
\ No newline at end of file
+Events can be grouped according to location or categories and users can also search events based on the same.
+
+##Getting started with the app##
```

Comment:
Use correct markdown such as # Getting started with the App. Revert for the rest.

Reply:
Good work .The commit messages are good.